src/main.js

```diff
@@ -1,6 +1,9 @@
 import Vue from 'vue'
 import App from './App.vue'
 
+
+
+
```

Comment:
@ChoudharyHamza extra blank lines here

Reply:
ok have removed the spaces and will avoid them